prow/cluster/starter/prowjob.tpr.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: extensions/v1beta1
+kind: ThirdPartyResource
```

Comment:
If this is an example I suggest switching to CRDs. They work in 1.7. We have one [here](https://github.com/openshift/release/blob/master/cluster/ci/config/prow/prow_crd.yaml).

Reply:
Again from `starter.yaml`

Reply:
I think we are on CRD now cc @cjwagner